src/frontend/test_runner/tests/testdata/struct_query.yaml

```diff
@@ -315,3 +315,58 @@
           string address = 1;
           string zipcode = 3;
         }
+- sql: |
+    insert into s values (1,2,(1,2,(1,2,3)));
+  logical_plan: |
+    LogicalInsert { table: s }
+      LogicalValues { rows: [[1:Int32, 2:Int32, {Some(Int32(1)), Some(Int32(2)), Some(Struct(StructValue { fields: [Some(Int32(1)), Some(Int32(2)), Some(Int32(3))] }))}:Struct { fields: [Int32, Int32, Struct { fields: [Int32, Int32, Int32] }] }]], schema: Schema { fields: [:Int32, :Int32, :Struct { fields: [Int32, Int32, Struct { fields: [Int32, Int32, Int32] }] }] } }
+  create_source:
+    row_format: protobuf
+    name: s
+    materialized: true
+    file: |
+      syntax = "proto3";
+      package test;
+      message TestRecord {
+        int32 v1 = 1;
+        int32 v2 = 2;
+        V v3 = 3;
+      }
+      message V {
+        int32 v1 = 1;
+        int32 v2 = 2;
+        U v3 = 3;
+      }
+      message U {
+        int32 v1 = 1;
+        int32 v2 = 2;
+        int32 v3 = 3;
+      }
+- sql: |
+    select * from s where s.v3 = (1,2,(1,2,3));
+  logical_plan: |
+    LogicalProject { exprs: [$1, $2, $3], expr_alias: [v1, v2, v3] }
+      LogicalFilter { predicate: ($3 = {Some(Int32(1)), Some(Int32(2)), Some(Struct(StructValue { fields: [Some(Int32(1)), Some(Int32(2)), Some(Int32(3))] }))}:Struct { fields: [Int32, Int32, Struct { fields: [Int32, Int32, Int32] }] }) }
```

Comment:
I wonder why the test format of StructValue remains the verbose form even after you impl fmt::Display in concise form?

Reply:
Maybe because the display of Datum is like these Some(...), so this looks like complex.